### ISSUE TITLE
Feature request: passing print statements through to verilator.

### ISSUE THREAD

**Issue by rdaly525** [leonardt/fault#48]
More debug features from python. One example is I would like to just stage a print statement.

for example:
```
tester.print("SomeArbitraryString")
tester.print(mysig1)

```

**Comment by rsetaluri** [comment]
Yeah we want to do this. In the first pass we simply did `print(port, format_str)` but I want to do the following (from #28 )

> Wondering if we can rename this to like `Print(format_str, *args)` where we have some special syntax for a port (like `%v` or something else that's not used). The target functionality is the same, it just makes it more clear that this is just a print statement, not returning you the value inline as a `peek` normally would.
> 
> I.e. `val = peek(foo.O)` doesn't make sense here, though that's the normal usage of peek. Then if we have a non-compiled tester in the future that's running in real time, we can actually have a `peek` that returns a value.
